es_ES/design-principles.md

```diff
@@ -0,0 +1,241 @@
+
+Se puede escuchar a los miembros del equipo referirse a "licking the cookie" (lamer la galleta). Asignarse un error a sí mismo, o indicar que se va a trabajar en él, les dice a otros miembros del equipo que no lo solucionen. Si entonces no trabajas en ello inmediatamente, estás actuando como alguien que ha tomado una galleta, la ha lamido para no atraer a otras personas, y luego no se la ha comido de inmediato. Por extensión, “unlicking the cookie” (deslamer la galleta) significa indicar al resto del equipo que no vas a trabajar en el error. inmediatamente después de todo, por ejemplo, desasignándote el error a ti mismo.
+
+Archiva los errores de cualquier cosa que encuentres que necesite hacer. Cuando implementa algo pero sabes que no está completo, archiva los errores por lo que no has hecho. De esa manera, podemos seguir la pista de lo que todavía hay que hacer.
+
+
+Regresiónes
+-----------
+
+Si un check-in ha causado una regresión en el tablero, retroceda el check-in (incluso si no es el suyo) a menos que hacerlo lleve más tiempo que arreglar el error. Cuando el enlace se rompe, ralentiza a todos los demás en el proyecto.
+
+
+Si las cosas se rompen, la prioridad de todos en el equipo debe ser ayudar al equipo a solucionar el problema. Alguien debe ser el dueño del issue y puede delegar responsabilidades a otros miembros del equipo. Una vez resuelto el problema, escribe un
+[post-mortem](https://github.com/flutter/flutter/wiki/Postmortems).
+Las post-mortem son para documentar lo que salió mal y cómo evitar que el problema (y toda la clase de problemas como éste) se repita en el futuro. Los post-mortem enfáticamante no son sobre la asignación de culpas.
+
+No hay nada vergonzoso en cometer errores.
+
+
+Preguntas
+---------
+
+Siempre está bien hacer preguntas. Nuestros sistemas son grandes, nadie será un experto en todos los sistemas.
+
+
+Resolución de conflictos
+-------------------
+
+Cuando varios colaboradores no están de acuerdo en la dirección de un parche en particular o en la dirección general del proyecto, el conflicto debe resolverse mediante la comunicación. Las personas que no están de acuerdo deben reunirse, tratar de entender los puntos de vista de los demás y trabajar para encontrar un diseño que responda a las preocupaciones de todos.
+
+Por lo general, esto es suficiente para resolver los problemas. Si no puede llegar a un acuerdo, pida consejo a un miembro más veterano del equipo.
+
+Desconfíe del acuerdo por desgaste, en el que una persona argumenta un punto repetidamente hasta que otros participantes se rinden en interés de seguir adelante. No se trata de una resolución de conflictos, ya que no aborda las preocupaciones de todos. Desconfíe del acuerdo por compromiso, en el que dos buenas soluciones competidoras se fusionan en una solución mediocre. Un conflicto se aborda cuando los participantes están de acuerdo en que la solución final es _mejor_ que todas las propuestas en conflicto. 
```

Comment:
Desconfía del acuerdo por agotamiento, en el que una persona argumenta un punto repetidamente hasta que otros participantes se rinden en interés de seguir adelante. No se trata de una resolución de conflictos, ya que no aborda las preocupaciones de todos. Desconfía del acuerdo por compromiso, en el que dos buenas soluciones competidoras se fusionan en una solución mediocre. Un conflicto se resuelve cuando los participantes están de acuerdo en que la solución final es mejor que todas las propuestas en conflicto. A veces la solución requiere más trabajo que cualquiera de las otras propuestas. Por favor, ve los comentarios anteriores donde introducimos la frase  “embrace the yak shave”.